tapir/utils/models.py

```diff
@@ -385,3 +388,13 @@ def copy_user_info(source, target):
     target.city = source.city
     target.country = source.country
     target.preferred_language = source.preferred_language
+
+
+def safe_redirect(redirect_url, request):
```

Comment:
(nit) Please leave a not where you copy-pasted this from.

And maybe create separate file for this? It doesn't really belong in modules. `utils/shortcuts.py` to mimic the django module name maybe?

Reply:
It's not directly copy-pasted, it's a mix of this SonarQube rule : https://rules.sonarsource.com/python/RSPEC-5146, a bit of googling for how to do without the DOMAINS_WHITELIST from the rule, then with the default parameter separate so that we can fit django models in it. 

I'll move it to a shortcuts.py tonight.